### PR TITLE
Adjusted Call Now modal to match the designs (Issue #307)

### LINKED ISSUE #307: Adjust Call Now modal to match the designs
Here's our call now modal: 

screen shot 2016-05-29 at 11 14 38 am

And here's the design:

screen shot 2016-05-29 at 11 15 04 am

Note the slew of differences, such as the size of it, the spacing, and the colors. Let's adjust our modal (located in `app/views/calls/_new_call.html.erb`) to match the designs.

### PR DESCRIPTION
**Pull request by dfelsent**
Adjusted Call Now modal per issue #307 specs. Note: all newly added styles are located in calls.scss.

Fixes #307

### CODE REVIEW

**Comment by colinxfleming** [comment]
Hey @dfelsent ! Looks like we're running into merge conflicts on this, can you pull down recent changes from the repo? (Try running `git pull https://github.com/colinxfleming/dcaf_case_management master` to pull down from upstream, if memory serves.)

File: app/assets/stylesheets/calls.scss [hunk 1]
```diff
@@ -1,3 +1,26 @@
 // Place all the styles related to the calls controller here.
 // They will automatically be included in application.css.
-// You can use Sass (SCSS) here: http://sass-lang.com/
\ No newline at end of file
+// You can use Sass (SCSS) here: http://sass-lang.com/
+
+div.calls-layout {
```
Comment (colinxfleming) [inline app/assets/stylesheets/calls.scss]: 💯

File: app/views/calls/_new_call.html.erb [hunk 2]
```diff
@@ -2,15 +2,19 @@
...
-        <%= link_to "I left a voicemail for the patient", pregnancy_calls_path(p, call: { status: "Left voicemail" } ), method: :post, remote: true %>
-        </br>
-        <%= link_to "I couldn't reach the patient", pregnancy_calls_path(p, call: { status: "Couldn't reach patient" } ), method: :post, remote: true %>
-      </div>
+       <h4 class="modal-title calls-request">Call <b><%= p.patient.name %></b> now:</h4>
+        <br>
+        <h4 class="calls-phone"><%= p.patient.primary_phone %></h4>
+        <br>
+        <%= button_to "I reached the patient", pregnancy_calls_path(p, call: { status: 'Reached patient'} ), method: :post, class: 'btn btn-primary', id: 'calls-btn' %>
+       <br><br>
```
Comment (colinxfleming) [inline app/views/calls/_new_call.html.erb]: @dfelsent I wonder if we're leaning a little too hard on `<br>` here -- thoughts on doing this using bootstrap classes rather than line breaks so it's cleaner? (If there's not an obvious path to it let's punt for now.)

**Comment by colinxfleming** [comment]
@dfelsent this looks great, thanks much! I had a question about our use of `br` tags, and we'll need to resolve merge conflicts, but after that we should be good to merge. 

Posting the change for @mebates to weigh in: 

image

**Comment by dfelsent** [comment]
I thought I had resolved the merge conflict but I see that tests are failing here... I'm not sure why. :/ Sorry about this, it's my first go-round with conflicts.

(And yes, I agree with you about the `br`s, but I figure it's best to focus on the conflict first.)

File: app/views/calls/_new_call.html_BACKUP_7141.erb [hunk 3]
```diff
@@ -0,0 +1,31 @@
+  <div class="modal fade new-call" tabindex="-1" role="dialog" id="call-<%= p.patient.primary_phone %>">
```
Comment (colinxfleming) [inline app/views/calls/_new_call.html_BACKUP_7141.erb]: @dfelsent This file looks like an artifact from a text editor crash -- can you remove it? Thanks much!

File: app/views/calls/_new_call.html_BASE_7141.erb [hunk 4]
```diff
@@ -0,0 +1,16 @@
+<div class="modal fade new-call" tabindex="-1" role="dialog" id="call-<%= p.patient.primary_phone %>">
```
Comment (colinxfleming) [inline app/views/calls/_new_call.html_BASE_7141.erb]: Same deal for this -- not familiar with files that are named like this, so please remove (but let me know if this serves a purpose I'm not familiar with!

File: app/views/calls/_new_call.html_LOCAL_7141.erb [hunk 5]
```diff
@@ -0,0 +1,20 @@
+<div class="modal fade new-call" tabindex="-1" role="dialog" id="call-<%= p.patient.primary_phone %>">
```
Comment (colinxfleming) [inline app/views/calls/_new_call.html_LOCAL_7141.erb]: Same deal for this -- not familiar with files that are named like this, so please remove (but let me know if this serves a purpose I'm not familiar with!

File: app/views/calls/_new_call.html_REMOTE_7141.erb [hunk 6]
```diff
@@ -0,0 +1,16 @@
+  <div class="modal fade new-call" tabindex="-1" role="dialog" id="call-<%= p.patient.primary_phone %>">
```
Comment (colinxfleming) [inline app/views/calls/_new_call.html_REMOTE_7141.erb]: Same deal for this -- not familiar with files that are named like this, so please remove (but let me know if this serves a purpose I'm not familiar with!

File: app/views/calls/_new_call.html.erb [hunk 7]
```diff
@@ -2,15 +2,19 @@
...
-        </br>
-        <%= link_to "I left a voicemail for the patient", pregnancy_calls_path(p, call: { status: "Left voicemail" } ), method: :post, remote: true %>
-        </br>
-        <%= link_to "I couldn't reach the patient", pregnancy_calls_path(p, call: { status: "Couldn't reach patient" } ), method: :post, remote: true %>
-      </div>
+       <h4 class="modal-title calls-request">Call <b><%= p.patient.name %></b> now:</h4>
+        <br>
+        <h4 class="calls-phone"><%= p.patient.primary_phone %></h4>
+        <br>
+        <%= button_to "I reached the patient", pregnancy_calls_path(p, call: { status: 'Reached patient'} ), method: :post, class: 'btn btn-primary', id: 'calls-btn' %>
```
Comment (colinxfleming) [inline app/views/calls/_new_call.html.erb]: We ran into a weird bug using `button_to` here -- for some reason, the server calls don't fire on `button_to` but they do on `link_to`, which in turn is making a couple of our tests fail. 

tl;dr: Let's change this back to `link_to` for now?

**Comment by dfelsent** [comment]
Just pushed up a change -- should be all fixed now!

**Comment by colinxfleming** [comment]
Perfect! Thank you for revising so quickly @dfelsent !
